sequoia/settings/rl/incremental/setting.py

```diff
@@ -479,6 +479,33 @@ def __post_init__(self):
         # FIXME: Really annoying little bugs with these three arguments!
         # self.nb_tasks = self.max_steps // self.steps_per_task
 
+        # If dataset has been wrapped, access the spec attribute of the dataset's venv attribute
+        # (Needed for Stable Baselines 3 wrapper VecTransposeImage)
+        if not hasattr(self.test_dataset, "spec") and hasattr(self.test_dataset, "venv") and hasattr(
+                self.test_dataset.venv, "spec"):
+            self.test_dataset.spec = self.test_dataset.venv.spec
+
+    def reset(self):
```

Comment:
Very intriguing! You're making the Setting into an environment? Where is this used? You're probably passing the setting to an algo by mistake, rather than the `train_env`. Should remove this `reset` and `step`, and if you need any of the logic here, then put it in a new Wrapper (although I'm pretty sure the StableBaselines3Method already adds wrappers to do the same as what you're doing here.)